Documentation/configuration.md

```diff
@@ -270,3 +270,38 @@ and password `gle`.
 Note that _within_ a particular configuration directory (either system or
 local), it is a syntax error for the same Docker registry to be defined in
 multiple files.
+
+
+
+### rktKind: `directories`
+
+This kind of configuration is used to customize the various directories that
+rkt uses. The configuration files should be placed inside a `directories`
+subdirectory (e.g. in `/usr/lib/rkt/directories` or
+`/etc/rkt/directories`).
+
+#### rktVersion: `v1`
+
+##### Description and examples
+
+The configuration works by mapping types of directories to their path in the
+file system. At the moment, the only configurable directory is the data
+directory, where container image data is stored to.
+
+For example, to store container images in your home partition instead of the
+root partition:
+
+`/etc/rkt/directories/paths.json`
+```js
+{
+	"rktKind": "directories",
+	"rktVersion": "v1",
+	"data": "/home/me/rkt"
+}
+```
+
+##### Override semantics
+
+Overriding is done for each directory. Not specifying a directory leaves it as
```

Comment:
The part about default values should be rather in `data` field description. Maybe you could mention `--dir` parameter's precendence over configuration files. `Overriding semantics` should describe what gets overridden when having configuration setting up the same things in both system and local directories. Examples would be nice here as well, though they are not going to be impressive for now.